deployment/playground/src/search.py

```diff
@@ -87,7 +87,7 @@ def search(
     if endpoint == 'suggestion':
         data[attribute_name] = attribute_value
     elif endpoint == 'search':
-        data['query'] = {'search_field': {attribute_name: attribute_value}}
+        data['query'] = {f'query_{attribute_name}': {attribute_name: attribute_value}}
```

Comment:
Is it possible to remove some nesting here? For example:
```suggestion
        data['query'] = {attribute_name: attribute_value}
```

Reply:
No, we unfortunately need this nesting as we now introduce the API to have multiple query fields.
So in the future, it might look like
```python
{'query': 
    {'title': {'text': 'a title'}},
    {'banner': {'uri': 'www.website/banner.jpg'}}
}
```